src/components/VideoControls/VideoControls.css

```diff
@@ -8,6 +8,9 @@
   left: 50%;
   transform: translate(-50%, -50%);
 }
+.controls:-webkit-fullscreen{
+  display: none;
```

Comment:
is this change needed? Or was it committed by accident? 🤔

Reply:
Yes, it was committed by accident, I'd delete it